src/client/containers/QuizPage/QuizPage.component.js

```diff
@@ -1,3 +1,45 @@
 import React from 'react';
+import './QuizPage.styles.css';
+import PageHeader from '../../components/PageHeaderComponent/PageHeader';
+import QuestionComponent from '../../components/Question/QuestionComponent';
+import QuizAnswers from '../../components/QuizAnswers/QuizAnswers';
+import SideMenu from '../../components/sideMenubar/sideMenu';
+import Buttons from '../../components/Buttons/Buttons';
+import ProgressBar from '../../components/ProgressBar.components/ProgressBar';
+import image1 from '../../assets/images/questionBackgrounds/question1background.png';
 
-export const QuizPage = () => <section>Quiz Page</section>;
+export const QuizPage = () => {
+  return (
+    <div>
+      <div className="main">
+        <div className="header_page">
+          <PageHeader />
+        </div>
+        <div className="middle_page">
+          <div className="sidemenu_page">
+            <SideMenu highLigtItem={3} />
+          </div>
+          <div className="question_page">
+            <QuestionComponent
+              imageSrc={image1}
+              question="When visiting a website, what is that you are most interested in?"
+            />
+            <div className="quizans_page">
+              <QuizAnswers
+                isAgreementQuestion={false}
```

Comment:
Hi Charmi, If i choose isAgreementQuestion={true} here, it's showing agree or disagree options, but it looks like they require some css to align them properly. Have a look Once...

Reply:
Thanks @rajanibusani for reviewing my PR.. I tried AgreementQuestion={true}  but i think for now as per requirement this one looks ok..  Thanks again :)

Reply:
Thanks @sandhana14 for review my PR :)